static/scripts/chatLoggedin.js

```diff
@@ -1,29 +1,72 @@
-function loadChatClient(session = null) {
-	let roomId = '';
-	const matches = RegExp('/courses/([^/]+).*').exec(window.location.pathname);
-	if (matches && matches.length >= 2) {
-		roomId = matches[1];
+function findMatrixUserId(session = null) {
+	if (session) {
+		return session.userId;
+	}
+
+	return window.localStorage.getItem('mx_user_id');
+}
+
+function extractRoomTypeAndIdFromPath(path) {
+	const matches = RegExp('/(course|team)s/([0-9a-f]{24})')
```

Comment:
```suggestion
	const matches = RegExp('^/(course|team)s/([0-9a-f]{24})')
```

Reply:
globally define the scopes, where the messenger is active for (courses, teams, ...)